packages/lesswrong/components/posts/PostsPage/PostsPage.tsx

```diff
@@ -251,7 +251,9 @@ const PostsPage = ({post, refetch, classes}: {
       <CitationTags
         title={post.title}
         author={post.user?.displayName}
-        coauthors={post.coauthors?.map(({displayName}) => displayName)}
+        coauthors={post.coauthors?.
+          filter(({ _id }) => !postCoauthorIsPending(post, _id)).
+          map(({displayName}) => displayName)}
```

Comment:
```suggestion
        coauthors={post.coauthors
          ?.filter(({ _id }) => !postCoauthorIsPending(post, _id))
          .map(({displayName}) => displayName)}
```

I've always seen it the way I've put it, never seen it the way you've done it, and grepped the codebase and confirmed my impression.